### PR TITLE
add FAQ entry, remove custom branch from PR template page

### LINKED ISSUE #2663: clarify Custom Branch setting
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/docs/collaborate/git/pr-template#configure-custom-branches

@runleonarun side note -- it seems like this page is about PR templates -- should the custom branch information be moved elsewhere?

### What part(s) of the page would you like to see updated?

The custom branch section should include a quick explainer that the Custom Branch section of the environment settings can accept a branch name, a tag name, or a commit SHA to get the correct version of your code from your git provider

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dave-connors-3**
## What are you changing in this pull request and why?

Closes #2663

Right now the docs for using the custom branch settings in a dbt cloud environment are nested within docs for PR Templates. This should be moved into a more general environment configuration. It should also be updated to clarify what you can use in that setting, and what the effects are for development and deployment environments. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: How do I use the `Custom Branch` settings in a dbt Cloud Environment?
+description: "Use custom code from your repository"
+sidebar_label: 'Custom Branch settings'
+id: custom-branch-settings
+---
+dbt Cloud environments allow users to specify custom git settings to use different versions of their dbt project repositories other than their default branch. After checking off the `Only run on a custom branch` checkbox in the `General Settings` section of the environment settings page, users can specify the particular **branch name or tag** in the`Custom Branch` setting that dbt will use when executing in that environment. Depending on the type of environment, this setting will have slightly different effects.
+
+## In development
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
## Development
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+In a development environment, dbt Cloud will reference the default branch in connected repositories (most commonly the `main` branch) in the IDE as the read only brnach that developers will create development branches from. The `Custom Branch` setting overrides that behavior if there is another branch that should be used. Users will not be able to make commits to the branch specified by this setting.  
+
+For example, you can use the `develop` branch of a connected repository. Edit an environment, then in "General settings" select  **Only run on a custom branch** , and in "Custom branch" type **develop** or the name of your custom branch.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/dev-environment-custom-branch.png" title="Configuring a custom base repository branch"/>
+
+## In deployment
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
## Deployment
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: How do I use the `Custom Branch` settings in a dbt Cloud Environment?
+description: "Use custom code from your repository"
+sidebar_label: 'Custom Branch settings'
+id: custom-branch-settings
+---
+dbt Cloud environments allow users to specify custom git settings to use different versions of their dbt project repositories other than their default branch. After checking off the `Only run on a custom branch` checkbox in the `General Settings` section of the environment settings page, users can specify the particular **branch name or tag** in the`Custom Branch` setting that dbt will use when executing in that environment. Depending on the type of environment, this setting will have slightly different effects.
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion

In dbt Cloud environments, you can change your git settings to use a different branch in your dbt project repositories besides the default branch. When you make this change, you run dbt on a custom branch. When specified, dbt Cloud executes models using the custom branch setting for that environment. Development and deployment environments have slightly different effects.

To specify a custom branch:
1. Edit an existing environment or create a new one
2. Select **Only run on a custom branch** under General Settings
3. Specify the **branch name or tag**

```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## In development
+
+In a development environment, dbt Cloud will reference the default branch in connected repositories (most commonly the `main` branch) in the IDE as the read only brnach that developers will create development branches from. The `Custom Branch` setting overrides that behavior if there is another branch that should be used. Users will not be able to make commits to the branch specified by this setting.
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
In a development environment, the default branch (commonly the `main` branch) is a read-only branch found in the IDE's connected repositories, which you can use to create development branches. Identifying a custom branch overrides this default behavior. Instead, your custom branch becomes read-only and can be used to create development branches. You will no longer be able to make commits to the custom branch. 
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+---
+dbt Cloud environments allow users to specify custom git settings to use different versions of their dbt project repositories other than their default branch. After checking off the `Only run on a custom branch` checkbox in the `General Settings` section of the environment settings page, users can specify the particular **branch name or tag** in the`Custom Branch` setting that dbt will use when executing in that environment. Depending on the type of environment, this setting will have slightly different effects.
+
+## In development
+
+In a development environment, dbt Cloud will reference the default branch in connected repositories (most commonly the `main` branch) in the IDE as the read only brnach that developers will create development branches from. The `Custom Branch` setting overrides that behavior if there is another branch that should be used. Users will not be able to make commits to the branch specified by this setting.  
+
+For example, you can use the `develop` branch of a connected repository. Edit an environment, then in "General settings" select  **Only run on a custom branch** , and in "Custom branch" type **develop** or the name of your custom branch.
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
For example, you can use the `develop` branch of a connected repository. Edit an environment, select  **Only run on a custom branch** in **General settings** , enter **develop** as the name of your custom branch.
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/dev-environment-custom-branch.png" title="Configuring a custom base repository branch"/>
+
+## In deployment
+
+When running jobs in a deployment environment, dbt will clone your project from your connected repository before executing your models. By default, dbt will pull from the default branch of your repository (most commonly the `main` branch). To control which version of your project is executed during job runs for a particular environment you can edit the `Custom Branch` setting as above to specify a different version of the project. Accepted values for this setting are **a branch name or a tag name.**
```
Comment (runleonarun) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
When running jobs in a deployment environment, dbt will clone your project from your connected repository before executing your models. By default, dbt uses the default branch of your repository (commonly the `main` branch). To specify a different version of your project for dbt to execute during job runs in a particular environment, you can edit the Custom Branch setting as shown in the previous steps. 
```

**Review by runleonarun — DISMISSED** [review]
@dave-connors-3 Nice work with this PR! 🙇🏻‍♀️ Thanks for getting it opened. I made some wording suggestions and used steps to help illustrate the process, and break up the page a little.

**Review by mirnawong1 — APPROVED** [review]
approving it for you @dave-connors-3 !

**Pull request merged**